core/urls.py

```diff
@@ -27,6 +27,7 @@ def limited_dataset_detail(request, slug, tablename):
     path("datasets/", views.dataset_list, name="dataset-list"),
     path("home/", views.home, name="home"),
     path("dataset/<slug>/", views.dataset_detail, name="dataset-detail"),
+    path("dataset/<slug>/_meta/", views.dataset_meta_detail, name="dataset-meta-detail"),
```

Comment:
Melhor mudar para `files`:
```suggestion
    path("dataset/<slug>/files/", views.dataset_meta_detail, name="dataset-meta-detail"),
```

Reply:
Resolvido em 181e21b